src/app/component/shared/items/items.module.ts

```diff
@@ -0,0 +1,19 @@
+import { CommonModule } from "@angular/common";
+import { NgModule } from "@angular/core";
+import { RouterModule } from "@angular/router";
+import { FontAwesomeModule } from "@fortawesome/angular-fontawesome";
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { fas } from "@fortawesome/free-solid-svg-icons";
+import { ItemComponent } from "./item/item.component";
+import { ItemsComponent } from "./items/items.component";
+
+@NgModule({
+  declarations: [ItemsComponent, ItemComponent],
+  imports: [CommonModule, RouterModule, FontAwesomeModule],
+  exports: [ItemsComponent, ItemComponent]
+})
+export class ItemsModule {
+  constructor() {
+    library.add(fas);
```

Comment:
This should be a function in the helper.ts file if possible. Otherwise fas should be grabbed from the helper.ts file.